src/content/docs/infrastructure/host-integrations/host-integrations-list/jmx-monitoring-install.mdx

```diff
@@ -10,28 +10,50 @@ redirects:
   - /docs/jmx-monitoring-install
   - /docs/jmx-integration-new-relic-infrastructure
   - /docs/infrastructure/host-integrations/host-integrations-list/jmx-monitoring-integration
+  - /docs/infrastructure/host-integrations/host-integrations-list/jmx-monitoring-configuration
 dataSource: jmx
 ---
 
-Our JMX [integration](/docs/integrations/host-integrations/getting-started/introduction-host-integrations) allows users to monitor any application that exposes metrics with JMX. The integration includes a default collection file that automatically collects key metrics from the JVM. You can also customize your metric collection with YAML files to collect any subset of metrics.
-
-Read on to install the integration, and to see what data we collect.
+Our JMX [integration](/docs/integrations/host-integrations/getting-started/introduction-host-integrations) allows users to monitor any application that exposes metrics with JMX. The integration includes a default collection file that automatically collects key metrics from the JVM. 
```

Comment:
"exposes metrics with JMX" is a little bit awkward. I think the idea here is that JMX is a monitoring ~thing for a Java app. so if you're already collecting metrics via jmx, you can port those metrics to New Relic. I might confirm what this looks like and massage out this sentence a little bit